tests/operators/test_agol_operators.py

```diff
@@ -0,0 +1,71 @@
+#!python3
+# test_agol_operators - test suite for AGOL operators
```

Comment:
Yup! 
Agreed that we should call these files "_operators" instead of "_operator".
Agreed that the `tests` folder structure should reflect the structure of what its testing.
Will do.